pkg/cache/cache.go

```diff
@@ -30,41 +32,68 @@ func New(cfg *config.Config) BundleCache {
 // FindBundle looks for a given bundle tag in the Porter bundle cache and
 // returns the path to the bundle if it exists. If it is not found, an
 // empty string and the boolean false value are returned.
-func (c *cache) FindBundle(bundleTag string) (string, bool, error) {
+func (c *cache) FindBundle(bundleTag string) (string, string, bool, error) {
 	bid := getBundleID(bundleTag)
 	bundleCnabDir, err := c.getCachedBundleCNABDir(bid)
 	cachedBundlePath := filepath.Join(bundleCnabDir, "bundle.json")
+	cachedReloPath := filepath.Join(bundleCnabDir, "relocation-mapping.json")
 	bExists, err := c.FileSystem.Exists(cachedBundlePath)
 	if err != nil {
-		return "", false, errors.Wrapf(err, "unable to read bundle %s at %s", bundleTag, cachedBundlePath)
+		return "", "", false, errors.Wrapf(err, "unable to read bundle %s at %s", bundleTag, cachedBundlePath)
 	}
 	if !bExists {
-		return "", false, nil
+		return "", "", false, nil
 	}
-	return cachedBundlePath, true, nil
+	//check for a relocation mapping next to it
+	rExists, err := c.FileSystem.Exists(cachedReloPath)
+	if err != nil {
+		return "", "", false, errors.Wrapf(err, "unable to read relocation mapping %s at %s", bundleTag, cachedReloPath)
+	}
+	if rExists {
+		return cachedBundlePath, cachedReloPath, true, nil
+	}
+	return cachedBundlePath, "", true, nil
 
 }
 
 // StoreBundle will write a given bundle to the bundle cache, in a location derived
 // from the bundleTag.
-func (c *cache) StoreBundle(bundleTag string, bun *bundle.Bundle) (string, error) {
+func (c *cache) StoreBundle(bundleTag string, bun *bundle.Bundle, reloMap relocation.ImageRelocationMap) (string, string, error) {
```

Comment:
nit: similar to `FindBundle`, I wonder if we should update the comment to mention a relocation mapping filepath may be returned?